server/server.js

```diff
@@ -40,10 +40,11 @@ Meteor.methods({
 
     var receiverEmail = PeopleCollection.findOne({ owner: receiver }).username;
 
-    if (senderUni !== null) {
+    if (senderUni != null) {
```

Comment:
Hm... could you document why you this fixes the problem? It seems pretty non-obvious.

Reply:
It doesn't but teammates agreed it was a better choice style-wise.

Reply:
Lol, that'd be why it's non obvious then 😆.